mne/tests/test_source_estimate.py

```diff
@@ -480,9 +480,8 @@ def attempt_assignment(stc, attr, val):
 def test_io_stc(tmp_path):
     """Test IO for STC files."""
     stc = _fake_stc()
-    stc.save(tmp_path / "tmp.stc")
-    stc2 = read_source_estimate(tmp_path / "tmp.stc")
-
+    stc.save(tmp_path / "tmp.stc", overwrite=True)
```

Comment:
is `overwrite=True` really needed here?  I wouldn't have expected it to be, since it's the first `.save` action in the test.